tensornetwork/backends/backend_test.py

```diff
@@ -113,6 +113,7 @@ def test_import_tensornetwork_without_backends():
 def test_basic_numpy_network_without_backends():
   #pylint: disable=import-outside-toplevel
   #pylint: disable=reimported
+  #pylint: disable=unused-variable
   import tensornetwork
```

Comment:
Why are we importing this here?

Reply:
No idea, I just disabled the linting error because it was annoying me

Reply:
ok. This should be fine then.